src/components/RecipeFind.vue

```diff
@@ -103,6 +104,33 @@ export default {
         params: { id: this.filterResults[0].filename },
       });
     },
+    onKeydown(event) {
+      const listElements = document.getElementsByClassName('list-group-item');
+      const currentHighlightedElement = document.getElementsByClassName('highlight')[0];
```

Comment:
Is it possible to select the elements within Vue? `this.$children` or by using `$refs`? I would like to keep element selection within the Vue instance where possible.

Reply:
I have used this.$children